plugins/contact-resources/src/components/Avatar.svelte

```diff
@@ -27,21 +27,29 @@
 </script>
 
 <script lang="ts">
-  import contact, { AvatarProvider, AvatarType } from '@hcengineering/contact'
+  import contact, { AvatarProvider, AvatarType, getAvatarColorForId } from '@hcengineering/contact'
   import { Client, Ref } from '@hcengineering/core'
   import { Asset, getResource } from '@hcengineering/platform'
   import { getBlobURL, getClient } from '@hcengineering/presentation'
   import { AnySvelteComponent, Icon, IconSize } from '@hcengineering/ui'
   import { getAvatarProviderId } from '../utils'
   import AvatarIcon from './icons/Avatar.svelte'
 
-  export let avatar: string | null | undefined = undefined
```

Comment:
better to have avatar and just name?: string and if passed parse it to abbreviature.